static/app/views/dashboards/widgetBuilder/buildSteps/filterResultsStep/releaseSearchBar.tsx

```diff
@@ -92,6 +93,11 @@ export function ReleaseSearchBar({
           searchSource="widget_builder"
           query={widgetQuery.conditions}
           savedSearchType={SavedSearchType.SESSION}
+          invalidMessages={{
```

Comment:
small nit, but I would probably rather this PR be named `feat(search-bar): Allow invalid reason customization`

and for this change to be made in a follow up PR

Reply:
hm I understand that but I think we can merge this small change here....it is like an example of this new functionality :)